src/components/Common.js

```diff
@@ -1,7 +1,7 @@
 import styled, { css } from 'styled-components'
 import { P } from './Typography'
 
-const screenBreakpoints = {
+export const screenBreakpoints = {
```

Comment:
let's move this to `src/utility/Constants.js` if its being used in multiple places

Reply:
Sounds good 👍 moved here 4559f62